### PR TITLE
Feat: Added Address and Checkout Page

### PR DESCRIPTION
**Pull request by sainath7878**
Added checkout page and address in checkout

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
The address and checkout page look good. You can also add payment integration and order summary.

File: src/context/product-context.js [hunk 1]
```diff
@@ -69,7 +68,8 @@ function ProductsProvider({ children }) {
                 toast.success("Item added to cart");
             }
         } catch (err) {
-            toast.error(err.response.data.errors[0]);
+            console.log(err);
+            // toast.error(err.response.data.errors[0]);
```
Comment (Maianki) [inline src/context/product-context.js]: you can remove the commented code.

File: src/server.js [hunk 2]
```diff
@@ -78,13 +101,22 @@ export function makeServer({ environment = "development" } = {}) {
         removeItemFromCartHandler.bind(this)
       );
 
+      // coupons route (public)
+      this.get("/coupon", getAllCouponsHandler.bind(this));
+
       // wishlist routes (private)
```
Comment (Maianki) [inline src/server.js:107]: remove commented code

**Review by Maianki — APPROVED** [review]
Eveything looks good you can go ahead with the merge.

**Pull request merged**